### PR TITLE
Version bumps for 2.2.0 beta 1.

### PR DESCRIPTION
**Pull request by ocean90**

### CODE REVIEW

File: tests/phpunit/data/translation.po [hunk 1]
```diff
@@ -51,4 +51,4 @@ msgstr "аз & ти не сме &amp;"
 #. Me, myself & Irene
 msgctxt "with_comment"
 msgid "baba"
-msgstr "баба"
\ No newline at end of file
+msgstr "баба"
```
Comment (toolstack) [inline tests/phpunit/data/translation.po]: You need the final return on this line otherwise the output won't match (see failing test).

Comment (ocean90) [reply tests/phpunit/data/translation.po]: I don't see a final return here https://github.com/GlotPress/GlotPress-WP/blob/develop/tests/phpunit/data/translation.po#L54 🤔

Comment (ocean90) [reply tests/phpunit/data/translation.po]: Ah, there should be **no** final newline.

**Comment by toolstack** [comment]
I'm not sure we should include an .editorconfig, but if we do should it be a separate PR?

**Review by toolstack — APPROVED** [review]
:+1:

**Pull request merged**